python/import_loss_model.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (c) 2018, GEM Foundation.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.
+# If not, see <https://www.gnu.org/licenses/agpl.html>.
+#
+"""
+Support for importing a loss model into the Challenge Fund loss database
+"""
+
+import json
+import sys
+
+from loss_model import LossModel
+# , LossMap, LossCurveMap
+from cf_common import Contribution, License
+
+from django.db import connections
+from django.conf import settings
+
```

Comment:
Out of curiosity, why are using Django when there are no Django models here? The idea is to add a Django-based WebUI later on?

Reply:
I guess just to use it's DB connections abstraction layer (_not_ the ORM)

Reply:
Essentially just for connections, cursors and DB settings.  I do not want an ORM or a web UI at this point.